app/views/home/_bookkeeper_office_manager.html.erb

```diff
@@ -0,0 +1,66 @@
+<% #Add the 'display-open' h3 class to display as opened on load %>
+<h3 class="display-open"><span class="chevron-closed">&nbsp;</span>Bookkeeper/Office Manager</h3>
+<div>
+	<h5>Location</h5>
+	<p>Remote</p>
+	<h5>About Girl Develop It</h5>
+	<p>
+		Girl Develop It is a 501(c)3 nonprofit organization that provides affordable and judgment-free opportunities for adult women interested in learning web and software development. Through in-person classes and community support, Girl Develop It helps women of diverse backgrounds achieve their technology goals and build confidence in their careers and their everyday lives.
+	</p>
+	<h5>About this Role</h5>
+	<p>
+		Girl Develop It is seeking a highly organized, experienced full-time Bookkeeper/Office Manager to provide administrative support to the organization’s central operations and to support the organization’s financial functions including budget development, income tax preparation, and charitable and business registration filings and renewals. The ideal candidate will have advanced experience in financial management for a nonprofit with rapid growth and multiple revenue streams, demonstrated professionalism in communications, and strong organizational and time management skill with acute attention to detail. We envision an enthusiastic professional who exemplifies our values of stewardship, integrity, community, and wants to part of a growing organization — with understanding that it’s necessary to be flexible to thrive in a fast paced environment. Experience working remotely or on a distributed team is highly desired.
```

Comment:
Apostrophe in "organization's" and "it's" -- you'll notice the difference once you swap in the apostrophe in your editor...